src/core/styles/_constants.scss

```diff
@@ -128,3 +139,10 @@ $z-index-base: 0;
 $z-index-below: -1;
 $z-index-above: 1;
 $z-index-max: 1000;
+
+// Box-shadow
+$shadow-xs: 1px 1px 2px $color-gray-900;
+$shadow-s: 0 4px 8px 0 hsla(from $color-gray-900 h s l / $opacity-10);
+$shadow-m: 0 4px 10px 0 hsla(from $color-gray-900 h s l / $opacity-10);
+$shadow-l: 0 4px 12px 0 hsla(from $color-gray-900 h s l / $opacity-10);
```

Comment:
I propose to combine these values
`0 4px 10px 0 hsla(from $color-gray-900 h s l / $opacity-10);` --> `0 4px 12px 0 hsla(from $color-gray-900 h s l / $opacity-10);`